python/neutron-understack/pyproject.toml

```
@@ -1,10 +1,35 @@
+[tool.poetry]
+name = "neutron-understack"
+version = "0.1.0"
+description = "Undercloud's ML2 driver for neutron"
+authors = ["Understack Developers"]
+readme = "README.rst"
 
 [tool.ruff]
 target-version = "py310"
 fix = true
+[tool.poetry.dependencies]
+python = "^3.10"
+requests = "^2.32.3"
```

do we need that specific version?
perhaps
```suggestion
requests = "^2"
neutron-lib = "^3"
```

would be more flexible

---

ok, changed in 34b7928